requirements-docs.txt

```diff
@@ -1 +1,3 @@
 Sphinx==1.3.1
+sphinx-django-command
```

Comment:
I'll have a look at it! Thanks and congrats :) In other django-apps, I'm using a dummy django project to have the django stack loaded such that for instances `models.py` can be introspected, for instance here: https://github.com/learningequality/kolibri/blob/master/docs/conf.py#L37